### PR TITLE
feat(chartjs): add chart.js doc

### PR DESCRIPTION
**Pull request by btourman**

### CODE REVIEW

**Comment by FGRibreau** [comment]
- [ ] Possible d'ajouter un lien "Chart.js" avant "References" ici https://github.com/image-charts/documentation/blob/master/mkdocs.yml#L32 ?
- [ ] je me demande s'il ne faudrait pas appeler le doc "chart.js.md" ?
- [ ] la section sur "icac" et "icretina" devrait être dans un bloc `!!!`:

```
!!! info "Enterprise+ subscription only"
    Retina support is only enabled in **Enterprise+** subscriptions
```


- [ ] le rendu pourrait être amélioré https://deploy-preview-22--image-charts-documentation.netlify.app/chartjs/

- [ ] il y a une erreur sur ce graphique: 


```
https://image-charts.com/chart.js/2.8.0?bkg=white&c=%7B%22type%22%3A%22bar%22%2C%22data%22%3A%7B%22labels%22%3A%5B%22January%22%2C%22February%22%2C%22March%22%2C%22April%22%2C%22May%22%2C%22June%22%2C%22July%22%5D%2C%22datasets%22%3A%5B%7B%22backgroundColor%22%3A%22rgba(54%2C162%2C235%2C0.5)%22%2C%22borderColor%22%3A%22rgb(54%2C162%2C235)%22%2C%22borderWidth%22%3A1%2C%22data%22%3A%5B73%2C41%2C29%2C61%2C-65%2C59%2C38%5D%7D%5D%7D%2C%22options%22%3A%7B%22legend%22%3A%7B%22display%22%3Afalse%7D%7D%7D&width=300&height=200)%02wzxhzdk:
```

File: docs/chart.js.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Chart.js
+
+This section describes how to use [chart.js](https://www.chartjs.org/docs/latest/charts/) Image Chart API.
+
+!!! info "Enterprise+ subscription only"
+    [icac](/enterprise) works with this configuration
+
+## Overview
+
+Image Charts is compatible with Chart.js You can put your Chart.js definition directly in URL like : `https://localhost:8080/chart.js/2.8.0?c={"type": "bar","data": {"labels": ["January","February","March"],"datasets": [{"data": [-31,-70,-30]}]},"options": {"legend": {"display": false}}}`
```
Comment (FGRibreau) [inline docs/chart.js.md]: @btourman "localhost" quoi !

**Comment by btourman** [comment]
Mauvais copié collé

File: docs/chart.js.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+#### Syntax
+
+```
+width=300
+height=400
+```
+
+**Examples :**
+
+![chart](https://image-charts.com/chart.js/2.8.0?bkg=white&c=%7B"type"%3A"bar"%2C"data"%3A%7B"labels"%3A%5B"January"%2C"February"%2C"March"%2C"April"%2C"May"%2C"June"%2C"July"%5D%2C"datasets"%3A%5B%7B"backgroundColor"%3A"rgba(54%2C162%2C235%2C0.5)"%2C"borderColor"%3A"rgb(54%2C162%2C235)"%2C"borderWidth"%3A1%2C"data"%3A%5B73%2C41%2C29%2C61%2C-65%2C59%2C38%5D%7D%5D%7D%2C"options"%3A%7B"legend"%3A%7B"display"%3Afalse%7D%7D%7D&width=300&height=200)
```
Comment (FGRibreau) [inline docs/chart.js.md:85]: @btourman 

IC:
![https://image-charts.com/chart.js/2.8.0?bkg=white&c=%7B%22type%22%3A%22bar%22%2C%22data%22%3A%7B%22labels%22%3A%5B%22January%22%2C%22February%22%2C%22March%22%2C%22April%22%2C%22May%22%2C%22June%22%2C%22July%22%5D%2C%22datasets%22%3A%5B%7B%22backgroundColor%22%3A%22rgba(54%2C162%2C235%2C0.5)%22%2C%22borderColor%22%3A%22rgb(54%2C162%2C235)%22%2C%22borderWidth%22%3A1%2C%22data%22%3A%5B73%2C41%2C29%2C61%2C-65%2C59%2C38%5D%7D%5D%7D%2C%22options%22%3A%7B%22legend%22%3A%7B%22display%22%3Afalse%7D%7D%7D&width=300&height=200]

vs

QuickChart:
![https://quickchart.io/chart?bkg=white&c=%7B%22type%22%3A%22bar%22%2C%22data%22%3A%7B%22labels%22%3A%5B%22January%22%2C%22February%22%2C%22March%22%2C%22April%22%2C%22May%22%2C%22June%22%2C%22July%22%5D%2C%22datasets%22%3A%5B%7B%22backgroundColor%22%3A%22rgba(54%2C162%2C235%2C0.5)%22%2C%22borderColor%22%3A%22rgb(54%2C162%2C235)%22%2C%22borderWidth%22%3A1%2C%22data%22%3A%5B73%2C41%2C29%2C61%2C-65%2C59%2C38%5D%7D%5D%7D%2C%22options%22%3A%7B%22legend%22%3A%7B%22display%22%3Afalse%7D%7D%7D&width=300&height=200]

**Pull request merged**